compiler/runtime/runtime.h

```diff
@@ -167,10 +168,12 @@ typedef struct {
   void* unused1;
   void* unused2;
 #endif
-  void (*m_onStateChange)(void*, long);
+  void* unused;
```

Comment:
Changing this dead field to unused padding is not the change I wanted to make. This PR started from investigations in the runtime system memory layout code, the copy from obstacks, etc. where there was unclear to me alignment, and this odd field. So far my attempts to remove this field, and to update `createGCType` in vtable.sk correspondingly, have failed. Even though removing this field preserves the needed 64-bit alignment, removing this field (or adding another one of padding), results in a skc binary that segfaults early. I suspect that somewhere else in the compiler or runtime system knows about the size of these vtable entries, but I have not found it. Any pointers welcome!

Reply:
@pikatchu tagging you in case you have any suggestions of where to look for other parts of the code that might rely on the size of vtable entries.